src/OpenDebugAD7/OpenDebug/Program.cs

```diff
@@ -65,7 +65,10 @@ private static int Main(string[] argv)
                         break;
                     case "--engineLogging":
                         loggingCategories.Add(LoggingCategory.EngineLogging);
-                        HostLogger.EnableHostLogging();
+                        HostLogger.InitalizeEngineLogger((logLevel, msg) =>
```

Comment:
I need to re-think this section since its breaking the tests.

Reply:
Did we do something like this somewhere before? I can't find code that would dump the engine logging to stdout.

I *think* what you want:
* Dump to std**err** here
* In the `--engineLogging=` case, only dump to the file.